manim/mobject/coordinate_systems.py

```diff
@@ -256,28 +265,89 @@ def create_axis(self, range_terms, axis_config, length):
         return axis
 
-    def coords_to_point(self, *coords):
+    def coords_to_point(self, *coords: Sequence[float]) -> np.ndarray:
+        """Transforms the vector formed from ``coords`` formed by the :class:`Axes`
+        into the corresponding vector with respect to the default basis.
+
+        Returns
+        -------
+        np.ndarray
+            A point that results from a change of basis from the coordinate system
+            defined by the :class:`Axes` to that of ``manim``'s default coordinate system
+        """
         origin = self.x_axis.number_to_point(0)
         result = np.array(origin)
         for axis, coord in zip(self.get_axes(), coords):
             result += axis.number_to_point(coord) - origin
         return result
 
-    def point_to_coords(self, point):
+    def point_to_coords(self, point: float) -> Tuple:
+        """Transforms the coordinates of the point which are with respect to ``manim``'s default
+        basis into the coordinates of that point with respect to the basis defined by :class:`Axes`.
+
+        Parameters
+        ----------
+        point
+            The point whose coordinates will be found.
+
+        Returns
+        -------
+        Tuple
+            Coordinates of the point with respect to :class:`Axes`'s basis
+        """
         return tuple([axis.point_to_number(point) for axis in self.get_axes()])
 
-    def get_axes(self):
+    def get_axes(self) -> VGroup:
+        """Gets the axes.
+
+        Returns
+        -------
+        :class:`~.VGroup`
+            A pair of axes.
+        """
         return self.axes
 
-    def get_coordinate_labels(self, x_values=None, y_values=None, **kwargs):
+    def get_coordinate_labels(
+        self,
+        x_values: Optional[Iterable[float]] = None,
+        y_values: Optional[Iterable[float]] = None,
+        **kwargs,
+    ) -> VDict:
+        """Gets labels for the coordinates
+
+        Parameters
+        ----------
+        x_values
+            Iterable of values along the x-axis, by default None.
+        y_values
+            Iterable of values along the y-axis, by default None.
+
+        Returns
+        -------
+        VDict
+            Labels for the x and y values.
+        """
         axes = self.get_axes()
         self.coordinate_labels = VGroup()
         for axis, values in zip(axes, [x_values, y_values]):
             labels = axis.add_numbers(values, **kwargs)
             self.coordinate_labels.add(labels)
         return self.coordinate_labels
 
-    def add_coordinates(self, x_values=None, y_values=None):
+    def add_coordinates(
+        self,
+        x_values: Optional[Iterable[float]] = None,
+        y_values: Optional[Iterable[float]] = None,
+    ):
+        """Adds the coordinates.
+
+        Parameters
+        ----------
+        x_values
+            Iterable of values along the x-axis, by default None.
+        y_values
+            Iterable of values along the y-axis, by default None.
```

Comment:
I don't think defaults should be mentioned unless they're hidden/ not accessible.
```suggestion
        x_values
            Iterable of values along the x-axis.
        y_values
            Iterable of values along the y-axis.
```

Reply:
https://numpydoc.readthedocs.io/en/latest/format.html#returns

Reply:
Defaults aren't mentioned anywhere else in the parameters section, plus, they're visible at the top of the method already.